### ISSUE TITLE
[BUG] Memory leak caused by Custom Docs plugin in silicon.util.custom_docs

### ISSUE THREAD

**Issue by oneseedfruit** [GodotNuts/GodotFirebase#169]
**Describe the bug**
The "Custom Docs" plugin in `addons/silicon.util.custom_docs` causes memory leak.

**To Reproduce**
Just enable the "Custom Docs" plugin in the project settings.

**Expected behavior**
Should not increase RAM usage every second.

**Environment:**
- OS: tried on Manjaro and Windows 10

**Comment by WolfgangSenff** [comment]
Hm. That's bad. At the moment this is kind of low priority because you just don't have to turn on that particular plugin, but I understand the leak sucks. The problem is the implementer of said plugin is currently without a computer, and we're waiting on a few tickets for him until he's able to get a new one. I'll leave this open to track it, but for now the advice is just don't turn it on and try to refer to the wiki - it's not perfect, but at least it shouldn't leak. :P

**Comment by oneseedfruit** [comment]
Alright, thanks! I'll do as you say. And thanks for making this plugin!!

**Comment by WolfgangSenff** [comment]
I'm closing this for now, as we have commit a change to remove the plugin from the repo. It seems there's a built-in bit of Godot that may be handling this in the future. We are only leaving the guts of it in (but enabling it won't do anything at this point) so that it can be "removed" from projects over time by disabling them.
